cluster/v1beta1/types_placementdecision.go

```diff
@@ -28,7 +28,12 @@ type PlacementDecision struct {
 
 // The placementDecsion label name holding the placement name
 const (
+	// Placement owner name.
 	PlacementLabel string = "cluster.open-cluster-management.io/placement"
+	// decision group index.
+	DecisionGroupIndexLabel string = "cluster.open-cluster-management.io/decisiongroupindex"
+	// decision group label.
+	DecisionGroupLabel string = "cluster.open-cluster-management.io/decisiongrouplabel"
```

Comment:
I think we should allow user to customize the key/value instead of a fixe key

Reply:
better to set the key as OCM API constant so controllers can identify it.